spec/install/post_bundle_message_spec.rb

```diff
@@ -81,6 +81,22 @@
         expect(out).to include(bundle_complete_message)
       end
     end
+
+    describe "with misspelled or non-existent gem name" do
```

Comment:
The reason I put it in `install/post_bundle_message_spec` is because I really just wanted to spec a helpful error message, not that the Resolver is able to do its job.

I might be able to move it to the resolver spec, but I will have to study `spec/support/indexes` for the example Gemfile that I provided.